<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTestResultListDialog</class>
 <widget class="QDialog" name="CTestResultListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTestResultListDialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>0</y>
     <width>191</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_result">
      <property name="text">
       <string>test:0  failures:0  errors:0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_msg"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_all">
      <property name="text">
       <string>SelectAll</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_run">
      <property name="text">
       <string>RunSelected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_runAll">
      <property name="text">
       <string>RunAll</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_unblock">
      <property name="text">
       <string>UnblockMode</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
